jest.config.js

```diff
@@ -1,19 +1,11 @@
-const { pathsToModuleNameMapper } = require('ts-jest/utils');
+const { pathsToModuleNameMapper } = require('ts-jest');
 const path = require('path');
 const { mapValues } = require('lodash');
 const { compilerOptions } = require('./packages/tsconfig.test');
 
 const projectDefault = {
   moduleNameMapper: {
     ...mapValues(pathsToModuleNameMapper(compilerOptions.paths), v => path.resolve(path.join('packages', v))),
-
```

Comment:
this was added recently, but is no longer needed with the newer version of jest

Reply:
This is awesome!